google-sheets/1.0/modules/ROOT/pages/google-sheets-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+You can configure one of these sources to use with Google Sheets Connector:
+
+* *HTTP > Listener*
++
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler*
++
+Initiates a flow when a time-based condition is met
+
+Text for using HTTP Listener as an input source
+
+For example, to configure an HTTP listener, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
+. On the *General* tab, specify connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Google Sheets Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Google Sheets* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+Each Google Sheets operation has its own required fields in Anypoint Studio. For more information, see
+the xref:google-sheets-connector-reference.adoc[Google Sheets Connector Reference].
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it's best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Google Sheets system. Google Sheets Connector supports OAuth 2.0 authorization.
+
+To configure the global element for Google Sheets Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, configure the authentication for the connection as described in xxx.
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```

Comment:
Make sure you run the link checker the xref here is improperly formatted and will break the build. It should be: 
xref:connectors::introduction/...